api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/CardClientTest.java

```diff
@@ -74,4 +77,45 @@ void shouldThrowAnExceptionWhenPostCallReturnsError() {
         assertThat(exception.getMessage()).isEqualTo("Exception caught trying to call card service at: " + baseUri + "/v1/cards");
         verify(restTemplate).postForEntity(baseUri + "/v1/cards", cardRequest, CardResponse.class);
     }
+
+    @Test
+    void shouldCallDepositFunds() {
+        DepositFundsRequest request = DepositFundsRequest.builder().build();
```

Comment:
I think we should have data in the request objects, same with the tests below too. Probably needs a TestDataFactory.